HealthcareSystem/GraphicalEditorServer/Mappers/pubilc.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace GraphicalEditorServer.Mappers
+{
+    internal class pubilc
+    {
+    }
+}
```

Comment:
For what is this used?

Reply:
I guess it has been added by mistake, I have removed it from the code.